gateway-service/src/main/resources/application.yml

```diff
@@ -69,11 +69,11 @@ apiml:
         jwtInitializerTimeout: 5
         personalAccessToken:
             enabled: false
-        oAuth:
+        oidc:
```

Comment:
It is actually OAuth2 configuration. We call it OIDC, but the OpenID part is done by the client.

Reply:
Renaming stuff could be definitely changed later in another PR.